rsocket/internal/SetupResumeAcceptor.cpp

```diff
@@ -231,6 +224,13 @@ void SetupResumeAcceptor::accept(
     std::unique_ptr<DuplexConnection> connection,
     OnSetup onSetup,
     OnResume onResume) {
+  DCHECK(inOwnerThread());
+
+  if(closed_) {
+    DCHECK(false) << "can't accept after having closed_!";
```

Comment:
I'm not sure if I'd want this as a `DCHECK`...  We shouldn't blow up if someone does `close() -> accept()`.

I think it's better to silently return, or throw an exception.

Reply:
Makes sense